src/Services/Accounts/SecretAccountStore.vala

```diff
@@ -11,9 +11,10 @@ public class Tuba.SecretAccountStore : AccountStore {
 		schema_attributes = new GLib.HashTable<string,Secret.SchemaAttributeType> (str_hash, str_equal);
 		schema_attributes["login"] = Secret.SchemaAttributeType.STRING;
 		schema_attributes["version"] = Secret.SchemaAttributeType.STRING;
+		//  schema_attributes["appid"] = Secret.SchemaAttributeType.STRING;
 		schema = new Secret.Schema.newv (
 			Build.DOMAIN,
-			Secret.SchemaFlags.NONE,
+			Secret.SchemaFlags.DONT_MATCH_NAME,
```

Comment:
Maybe it is worthwhile to add a comment with a [reference to the gnome-keyring upstream issue](https://gitlab.gnome.org/GNOME/gnome-keyring/-/issues/89) here so that the reader knows why the `DONT_MATCH_NAME` is necessary?